pygmo/docstrings.cpp

```diff
@@ -2336,6 +2336,86 @@ See also the docs of the relevant C++ method :cpp:func:`pagmo::nsga2::get_log`.
 )";
 }
 
+std::string gaco_docstring()
+{
+    return R"(__init__(gen = 1, ker = 63u, q = 1.0, oracle = 0., acc = 0.01, threshold = 1u,
+fstop = 0.000000000001, n_gen_mark = 7u, impstop = 100000u, evalstop = 100000u, focus = 0.,
+paretomax = 10u, epsilon = 0.9, seed = random)
+
+Giacomo's Ant Colony Optimization algorithm (GACO).
```

Comment:
Use the same text as in the c++ doxygen strings. No reference to Giacomo, rather to the paper from Schueler.